examples/etcd/etcd-in-cluster/ocp/blueprint-v2/etcd-incluster-ocp-blueprint.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: etcd-blueprint
+actions:
+  backup:
+    outputArtifacts:
+      etcdBackup:
+        kopiaSnapshot: "{{ .Phases.uploadSnapshot.Output.kopiaOutput }}"
+    phases:
+    - func: KubeTask
+      name: takeSnapshot
+      args:
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.80.0
+        command:
+          - sh
+          - -o
+          - errexit
+          - -o
+          - pipefail
+          - -c
+          - |
+            export endpoints="{{ index .Object.data "endpoints" | toString | b64dec }}"
+            export labels="{{ index .Object.data "labels" | toString | b64dec }}"
+            export etcdns="{{ index .Object.data "etcdns" | toString | b64dec }}"
+            # Get a member of etcd cluster
+            ETCD_POD=$(kubectl get pods -n $etcdns -l $labels -ojsonpath='{.items[0].metadata.name}')
+            # exec the snapshot save command
+            kubectl exec -it -n $etcdns $ETCD_POD -c etcd -- sh -c "ETCDCTL_ENDPOINTS=$endpoints etcdctl snapshot save /tmp/etcd-backup.db"
+            # this pod name will be used to copy and remove the snapshot
+            kando output etcdPod $ETCD_POD
+            kando output etcdNS $etcdns
+
+    - func: KubeTask
+      name: uploadSnapshot
+      args:
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.80.0
+        command:
+          - sh
+          - -o
+          - errexit
+          - -o
+          - pipefail
+          - -c
+          - |
+            BACKUP_LOCATION='etcd-backup.db.gz'
+            kubectl cp -c etcd {{ .Phases.takeSnapshot.Output.etcdNS }}/{{ .Phases.takeSnapshot.Output.etcdPod }}:/tmp/etcd-backup.db /tmp/etcd-backup.db
+            gzip -c /tmp/etcd-backup.db  | kando location push --profile '{{ toJson .Profile }}' --path "${BACKUP_LOCATION}" --output-name "kopiaOutput" -
+
+    - func: KubeTask
+      name: removeSnapshot
+      args:
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.80.0
+        command:
+          - sh
+          - -o
+          - errexit
+          - -o
+          - pipefail
+          - -c
+          - |
+            kubectl exec -it -n {{ .Phases.takeSnapshot.Output.etcdNS }} "{{ .Phases.takeSnapshot.Output.etcdPod }}" -c etcd -- sh -c "rm -rf  /tmp/etcd-backup.db"
+  restore:
+    inputArtifactNames:
+    - etcdBackup
+    phases:
```

Comment:
[nit] Understanding the restore workflow could be difficult to understand without knowing the context. Should we summarize the workflow in the comments?